src/main/java/racingcar/controller/RacingGameConsoleController.java

```diff
@@ -0,0 +1,59 @@
+package racingcar.controller;
+
+import java.io.IOException;
+
+import racingcar.domain.CarGroup;
+import racingcar.domain.RacingGame;
+import racingcar.domain.RacingResult;
+import racingcar.domain.RandomNumberGenerator;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class RacingGameConsoleController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public RacingGameConsoleController() {
+        this.inputView = new InputView();
+        this.outputView = new OutputView();
+    }
+
+    public void run() throws IOException {
```

Comment:
이번 미션에서 중요한 부분은 아니지만..

IOException이 Controller 까지 넘어와서 그냥 던져져야할 이유가 있을까요?

Reply:
계속해서 던져지는 메서드의 throws를 내부에서 try-catch로 수정했습니다.

```suggestion
    public void run() {
```

```java
    private CarGroup createCarGroup() {
        try {
            return new CarGroup(inputView.readCarNames());
        } catch (IllegalArgumentException e) {
            System.out.println(e.getMessage());
            return createCarGroup();
        }
    }
```